site/en/blog/new-in-chrome-93/index.md

```diff
@@ -0,0 +1,253 @@
+to the `document` or `shadowRoot` by calling `adoptedStyleSheets`.
+
+```js
+import sheet from './styles.css' assert { type: 'css' };
+document.adoptedStyleSheets = [sheet];
+shadowRoot.adoptedStyleSheets = [sheet];
+```
+
+But beware, if you leave off the `assert` - the file will be treated as
+JavaScript, and won't work!
+
+Check out [Using CSS Module Scripts to import stylesheets][wd-css-modules] on
+web.dev for complete details.
+
+## Multi-Screen Window Placement API {: #window-placement }
+
+For some apps, opening new windows, and putting them in specific places,
+or specific displays is an important feature. For example, when using Slides to
+present, I want the slides to appear full screen on the primary display, and
+my speaker notes to appear on the other display.
+
+The Multi-Screen Window Placement API makes it possible to enumerate the
+displays connected to the users machine, and place windows on specific screens.
+This is its second origin trial, and we've made a number of changes based on
+your feedback.
+
+You can quickly check if there's more than one screen connected to the device:
+
+```js
+const isExtended = window.screen.isExtended;
+// returns true/false
+```
+
+But, the key functionality is in `window.getScreens()`, which provides all the
+details  about the attached displays.
+
+```js
+const screens = await window.getScreens();
+// returns
+// {
+//    currentScreen: {...}
+//    oncurrentscreenchange: null
+//    onscreenschange: null
+//    screens: [{...}, {...}]
+// }
+```
+
+For example, you can determine the primary screen, then use `requestFullscreen`
+to display an element on that screen.
+
+```js
+try {
+  const screens = await window.getScreens();
+  const primary = screens.filter((screen) => screen.primary)[0];
+  await elem.requestFullscreen({ screen: primary });
+} catch (err) {
+  console.error(err);
+}
+```
+
+And it provides a way to listen for changes, for example if a new display is
+plugged in, or removed.
+
+```js
+const screens = await window.getScreens();
+let numScreens = screens.screens.length;
+screens.addEventListener('screenschange', (event) => {
+  if (screens.screens.length !== numScreens) {
+    console.log('Screen count changed');
+    numScreens = screens.screens.length;
+  }
+});
+```
+
+Check out Tom's article
+[Managing several displays with the Multi-Screen Window Placement API][wd-multi-screen]
+on web.dev for a deeper dive!
+
+## Shortened release cycle {:# shortened-release-cycle }
+
+In March, we announced our plans to [shorten the release cycle][dcc-shorter-cycle]
+and ship a new version of Chrome every 4 weeks.
+
+That time has arrived, and we'll ship Chrome 94 on September 21st. You can find
+planned release dates for each version on the [Chrome Calendar][chrome-cal].
+
+## New PWA features {: #new-pwa}
+
+If you're building a Progressive Web App, there are two new origin trials
+worth checking out.
+
+### URL handlers for PWAs {: #url-handlers }
```

Comment:
This got bumped from 93. It's currently slated for 94, but it's not clear that it's going out.

Reply:
Its in OT now, so I think it's fair to include.